worker/buildbot_worker/test/unit/test_scripts_base.py

```diff
@@ -33,7 +34,10 @@ class TestIsWorkerDir(misc.FileIOMixin, misc.StdoutAssertionsMixin,
 
     def setUp(self):
         # capture output to stdout
-        self.mocked_stdout = io.BytesIO()
+        if PY3:
```

Comment:
We can have a compat.py

buildbot_worker.compat.NativeStringIO
buildbot_worker.compat.enforceNative
buildbot_worker.compat.enforceString
buildbot_worker.compat.enforceBytes

That is what I meant by backporting.